### PR TITLE
cgroupsv2: failed to write on cpuset.cpus.exclusive

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OCPBUGS-34812

### CODE REVIEW

**Comment by mburke5678** [comment]
@haircommander PTAL

**Comment by mburke5678** [comment]
@sunilcio PTAL

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3118,6 +3118,8 @@ In the following tables, features are marked with the following statuses:
 
 * When a {product-title} {product-version} cluster is installed or configured as a postinstallation activity on a single VIOS host with virtual SCSI storage on {ibm-power-name} with multipath configured, the CoreOS nodes with multipath enabled fail to boot. This behavior is expected as only one path is available to the node. (link:https://issues.redhat.com/browse/OCPBUGS-32290[*OCPBUGS-32290*]).
 
+* When using CPU load balancing on cgroupv2, a pod can fail to be created if another pod that has access to exclusive CPUs already exists. This can happen when a pod is deleted and another one is quickly created to replace it. As a workaround, ensure that the old pod is fully terminated before attempting to create the new one. (link:https://issues.redhat.com/browse/OCPBUGS-34812[*OCPBUGS-34812*])
```
Comment (bergerhoffer) [inline release_notes/ocp-4-16-release-notes.adoc:3121]: Would "can fail to start" keep the same meaning? Just a suggestion to be more concise, if it works.

Comment (mburke5678) [reply release_notes/ocp-4-16-release-notes.adoc:3121]: @haircommander @sunilcio WDYT ^^

Comment (haircommander) [reply release_notes/ocp-4-16-release-notes.adoc:3121]: yeah it's just as correct (they're both slightly wrong but for this context I think it's okay)

**Pull request merged**